src/InfoColumn.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Image, View } from 'react-native';
+
+import InfoAccordion, { InfoAccordionProps } from './InfoAccordion';
+import Link from './Link';
+import Typography from './Typography';
+import { createStyles } from './theme';
+
+export interface InfoColumnProps {
+  icon?: string;
+  description?: string;
+  videoUrl?: string;
```

Comment:
nit: make these "Uri" instead of "Url", cause they could be ipfs/arweave URIs or something in the future